### PR TITLE
Replace packr with native go:embed

### PR DESCRIPTION
**Pull request by rfranzke**
**How to categorize this PR?**

/area dev-productivity
/kind cleanup
/platform equinix-metal

**What this PR does / why we need it**:
Meanwhile, we have updated to Golang 1.16 and can use the native `go:embed` feature which allows us to eliminate packr.

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,10 +8,10 @@ require (
 	github.com/coreos/go-systemd/v22 v22.1.0
 	github.com/frankban/quicktest v1.9.0 // indirect
 	github.com/gardener/etcd-druid v0.3.0
-	github.com/gardener/gardener v1.19.1-0.20210401122359-e16e6ec0002e
+	github.com/gardener/gardener v1.20.1
 	github.com/gardener/machine-controller-manager v0.36.0
 	github.com/go-logr/logr v0.3.0
-	github.com/gobuffalo/packr/v2 v2.8.1
+	github.com/gobuffalo/packr/v2 v2.5.1
```
Comment (kon-angelo) [inline go.mod:14]: packr is still in go.mod because there is a still reference in `hack/tools.go`

**Review by kon-angelo — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (rfranzke) [reply go.mod]: Thanks for catching!

**Review by kon-angelo — APPROVED** [review]
/lgtm

**Pull request merged**